docs/system-designer/designer-tasks.md

```diff
@@ -206,12 +202,11 @@ In the **block library** search box, enter a block type name.
   > * The **configured** tab contains blocks that have already been installed, named, and configured.
 
 1. Drag the block type to the canvas.
-1. In the **block name** box, enter a meaningful name and click **accept**.
-<img class="left border" src="/img/cloud/Hello-CreateBlockSimulate.png" width="250" />
+1. In the **name** box, enter a meaningful name and click **accept**.
 
 > **[info] Block names**
 >
-> Block names cannot be edited after creation.
+> Block names cannot be edited after creation in versions less than 3.0 of nio.
```

Comment:
let's switch this around to start "In nio versions less than 3.0, block names cannot be edited after creation."